hw/ip/aes/dv/aes_sim_cfg.hjson

```diff
@@ -36,16 +39,29 @@
   uvm_test: aes_base_test
   uvm_test_seq: aes_wake_up_vseq
 
+  // Update the default build mode to add options specific to AES C model compilation.
+  build_modes: [
+    {
+      name: default
```

Comment:
'default' is the basic compilation set (in case a DUT has more than one compilation sets). We are enabling the simulator tool-specific build options for compiling the aes_model.